doc/develop/getting_started/index.rst

```diff
@@ -523,12 +519,52 @@ that are used to emulate, flash and debug Zephyr applications.
             extracted under ``$HOME``, the resulting installation path will be
             ``$HOME/zephyr-sdk-0.14.2``.
 
+         .. code-block:: bash
+
+            tar xvf zephyr-sdk-0.14.2_linux-x86_64_minimal.tar.gz
+
+      #. Determine what architectures you intended to use with Zephyr:
+
+	 The SDK supports all architectures Zephyr targets, but users may also
+	 choose to install only a single architecture's toolchain if they do not
+	 intend to use Zephyr across multiple architectures. Below is a list of
+	 the toolchains supported by the SDK:
+
+	   * aarch64-zephyr-elf
+           * arc64-zephyr-elf
+           * arc-zephyr-elf
+           * arm-zephyr-eabi
+           * mips-zephyr-elf
+           * nios2-zephyr-elf
+           * riscv64-zephyr-elf
+           * sparc-zephyr-elf
+           * x86_64-zephyr-elf
+           * xtensa-espressif_esp32_zephyr-elf
+           * xtensa-espressif_esp32s2_zephyr-elf
+           * xtensa-intel_apl_adsp_zephyr-elf
+           * xtensa-intel_bdw_adsp_zephyr-elf
+           * xtensa-intel_byt_adsp_zephyr-elf
+           * xtensa-intel_s1000_zephyr-elf
+           * xtensa-nxp_imx_adsp_zephyr-elf
+           * xtensa-nxp_imx8m_adsp_zephyr-elf
+           * xtensa-sample_controller_zephyr-elf
+
+	 For example, a user who intends to only use Zephyr with ARM
+	 architectures could install the SDK with the following command:
+
+	 .. code-block:: bash
+
+		./setup.sh -h -c -t arm-zephyr-eabi
+
+	 If you are unsure which architecture you need, or would like to use
+	 multiple, substitute ``-t all`` for ``-t {toolchain_name}`` below.
```

Comment:
I think this is too much information for a GSG. I think that a better solution would probably be to create an online toolchain installer that asks which archs you want, if you want host tools, etc. then downloads just what is needed @stephanosio would that make sense?

Reply:
> an online toolchain installer that asks which archs you want, if you want host tools, etc. then downloads just what is needed

That is exactly what the minimal distribution bundle mentioned in https://github.com/zephyrproject-rtos/zephyr/pull/46949#discussion_r910096307 is.

Reply:
Yes, it supports both interactive (it asks if you want to install arch A, B, C, and so on) and non-interactive (command-line param controlled) mode.

You can try it out by downloading one here:
https://github.com/zephyrproject-rtos/sdk-ng/releases/tag/v0.14.2

> [1] Minimal bundle does not contain any toolchains and allows users to choose the toolchains to download and install.

Reply:
The problem with this has more to do with, for instance, "I want to build `esp32c3_devkitm`, which toolchain do I need? Is it `xtensa-espressif_esp32_zephyr-elf` or ` riscv64-zephyr-elf`?" (FYI, it is the latter).

For some, it may be quite obvious and straight-forward (e.g. `qemu_riscv64` requires `riscv64-zephyr-elf`); for others, not so much:

* `qemu_xtensa` requires `xtensa-sample_controller_zephyr-elf` (and not any other `xtensa-` toolchains)
* `fvp_baser_aemv8r` requires `aarch64-zephyr-elf` and not `arm-zephyr-eabi`
* `xenvm` requires `aarch64-zephyr-elf`
and the list goes on.

For the seasoned Zephyr developers and users, it is easy enough; not so much for the people who are just getting started with Zephyr and have no idea what "boards" and "SoC" really mean and/or how they are organised.

Reply:
> For the seasoned Zephyr developers and users, it is easy enough; not so much for the people who are just getting started with Zephyr and have no idea what "boards" and "SoC" really mean and/or how they are organised.

For someone new to embedded development, I agree that we should not guide them down this path. Maybe the best way to do that is recommending that developers install all toolchains by default.

However, I would expect any experienced embedded developer who is getting started with Zephyr to know the toolchain they need. I personally feel the getting started guide should at least inform developers that the minimal toolchain option exists.

Maybe the best way to approach this is to provide guidance to run the toolchain installer with `-t all`, and mention that it can be run interactively to only install specific toolchains. What are your thoughts on that approach?